docker/humlib.profiles

```diff
@@ -0,0 +1,5 @@
+export HUMLIB_BE_HOME=<YOUR PATH>
```

Comment:
Can we offer these as IntelliJ run configurations?

Reply:
how :o

Reply:
Let‘s just write an issue and do it later. It would only be convenience.